redirects/16.0.txt

```diff
@@ -54,27 +54,37 @@ applications/inventory_and_mrp/manufacturing/quality_control/quality_control_poi
 applications/inventory_and_mrp/inventory/management/lots_serial_numbers/expiration_dates.rst applications/inventory_and_mrp/inventory/product_management/product_tracking/expiration_dates.rst                                # /inventory_and_mrp/inventory/management/lots_serial_numbers/* -> /inventory_and_mrp/inventory/product_management/product_tracking/*
 applications/inventory_and_mrp/inventory/management/inventory_adjustments.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products.rst                                             # /inventory_and_mrp/inventory/management/inventory_adjustments -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products
 applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products.rst                              # /inventory_and_mrp/inventory/management/inventory_adjustments/count_products -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/*
 applications/inventory_and_mrp/inventory/management/warehouses.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst                                                                       # /inventory_and_mrp/inventory/management/warehouses -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management
 applications/inventory_and_mrp/inventory/management/warehouses/resupply_warehouses.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/resupply_warehouses.rst                               # /inventory_and_mrp/inventory/management/warehouses/* -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/*
+applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/resupply_warehouses.rst applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/resupply_warehouses.rst                    # /inventory_and_mrp/inventory/warehouses_storage/inventory_management/* -> /inventory_and_mrp/inventory/warehouses_storage/replenishment/*
 applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouse_replenishment_transfer.rst     # /inventory_and_mrp/inventory/management/warehouses/* -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/*
+applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouse_replenishment_transfer.rst applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/warehouse_replenishment_transfer.rst # /inventory_and_mrp/inventory/warehouses_storage/inventory_management/* -> /inventory_and_mrp/inventory/warehouses_storage/replenishment/*
```

Comment:
update comment :)
`/*` would mean the entire directory `inventory_management/` is being redirected, and that is not the case
```suggestion
applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouse_replenishment_transfer.rst applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/warehouse_replenishment_transfer.rst # /inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouse_replenishment_transfer -> /inventory_and_mrp/inventory/warehouses_storage/replenishment/warehouse_replenishment_transfer
```